libsolidity/lsp/LanguageServer.cpp

```diff
@@ -260,6 +260,10 @@ bool LanguageServer::run()
 			else
 				m_client.error({}, ErrorCode::ParseError, "\"method\" has to be a string.");
 		}
+		catch (RequestError const& error)
+		{
+			m_client.error(id, error.code(), boost::current_exception_diagnostic_information());
```

Comment:
AFAIK, `diagnostic_information` contains everything (line number, etc) and is rather difficult to understand for regular users. Shouldn't we just use `error.comment() ? *error.comment() : string{}`?